packages/react/package.json

```diff
@@ -7,6 +7,7 @@
   "main": "react.production.min.js",
   "scripts": {
     "build": "echo \"No build command installed\"",
+    "copy": "../../scripts/copy.js react @esm-bundle/react esm/react.production.min.*",
```

Comment:
Args:
1. monorepo package folder name eg.react 
2. dependency package name eg. @esm-bundle/react
3. path within the package folder in node_modules, eg. esm/react.production.min.*